packages/block-library/src/search/index.php

```diff
@@ -203,27 +213,17 @@ function register_block_core_search() {
 			'render_callback' => 'render_block_core_search',
 		)
 	);
-}
-add_action( 'init', 'register_block_core_search' );
 
-/**
- * Ensure that the view script has the `wp-interactivity` dependency.
- *
- * @since 6.4.0
- *
- * @global WP_Scripts $wp_scripts
- */
-function block_core_search_ensure_interactivity_dependency() {
```

Comment:
I don't remember either, to be honest. It seems it was included in this pull request in case it helps: [link](https://github.com/WordPress/gutenberg/pull/54594).

Reply:
Ok, I don't think it's needed then. Thanks, Mario 🙂